queue_services/entity-emailer/src/entity_emailer/config.py

```diff
@@ -20,131 +40,88 @@
-    ACCOUNT_SVC_CLIENT_SECRET = os.getenv('ACCOUNT_SVC_CLIENT_SECRET', None)
+    ACCOUNT_SVC_CLIENT_ID = os.getenv("ACCOUNT_SVC_CLIENT_ID", None)
+    ACCOUNT_SVC_CLIENT_SECRET = os.getenv("ACCOUNT_SVC_CLIENT_SECRET", None)
     # variables
-    LEGISLATIVE_TIMEZONE = os.getenv('LEGISLATIVE_TIMEZONE', 'America/Vancouver')
-    TEMPLATE_PATH = os.getenv('TEMPLATE_PATH', None)
+    LEGISLATIVE_TIMEZONE = os.getenv("LEGISLATIVE_TIMEZONE", "America/Vancouver")
+    TEMPLATE_PATH = os.getenv("TEMPLATE_PATH", None)
 
     SQLALCHEMY_TRACK_MODIFICATIONS = False
 
-    NAMEX_AUTH_SVC_URL = os.getenv('NAMEX_AUTH_SVC_URL', None)
-    NAMEX_SVC_URL = os.getenv('NAMEX_SVC_URL', None)
-    NAMEX_SERVICE_CLIENT_USERNAME = os.getenv('NAMEX_SERVICE_CLIENT_USERNAME', None)
-    NAMEX_SERVICE_CLIENT_SECRET = os.getenv('NAMEX_SERVICE_CLIENT_SECRET', None)
+    NAMEX_AUTH_SVC_URL = os.getenv("NAMEX_AUTH_SVC_URL", None)
+    NAMEX_SVC_URL = os.getenv("NAMEX_SVC_URL", None)
+    NAMEX_SERVICE_CLIENT_USERNAME = os.getenv("NAMEX_SERVICE_CLIENT_USERNAME", None)
+    NAMEX_SERVICE_CLIENT_SECRET = os.getenv("NAMEX_SERVICE_CLIENT_SECRET", None)
 
     # POSTGRESQL
-    DB_USER = os.getenv('DATABASE_USERNAME', '')
-    DB_PASSWORD = os.getenv('DATABASE_PASSWORD', '')
-    DB_NAME = os.getenv('DATABASE_NAME', '')
-    DB_HOST = os.getenv('DATABASE_HOST', '')
-    DB_PORT = os.getenv('DATABASE_PORT', '5432')
-    SQLALCHEMY_DATABASE_URI = 'postgresql://{user}:{password}@{host}:{port}/{name}'.format(
-        user=DB_USER,
-        password=DB_PASSWORD,
-        host=DB_HOST,
-        port=int(DB_PORT),
-        name=DB_NAME,
-    )
+    DB_USER = os.getenv("DATABASE_USERNAME", "")
+    DB_PASSWORD = os.getenv("DATABASE_PASSWORD", "")
+    DB_NAME = os.getenv("DATABASE_NAME", "")
+    DB_HOST = os.getenv("DATABASE_HOST", "")
+    DB_PORT = os.getenv("DATABASE_PORT", "5432")
+    if DB_UNIX_SOCKET := os.getenv("DATABASE_UNIX_SOCKET", None):
+        SQLALCHEMY_DATABASE_URI = f"postgresql+pg8000://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}?unix_sock={DB_UNIX_SOCKET}/.s.PGSQL.5432"
+    else:
+        SQLALCHEMY_DATABASE_URI = f"postgresql+pg8000://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
 
-    TRACKER_DB_USER = os.getenv('TRACKER_DATABASE_USERNAME', '')
-    TRACKER_DB_PASSWORD = os.getenv('TRACKER_DATABASE_PASSWORD', '')
-    TRACKER_DB_NAME = os.getenv('TRACKER_DATABASE_NAME', '')
-    TRACKER_DB_HOST = os.getenv('TRACKER_DATABASE_HOST', '')
-    TRACKER_DB_PORT = os.getenv('TRACKER_DATABASE_PORT', '5432')
-    TRACKER_SQLALCHEMY_DATABASE_URI = 'postgresql://{user}:{password}@{host}:{port}/{name}'.format(
-        user=TRACKER_DB_USER,
-        password=TRACKER_DB_PASSWORD,
-        host=TRACKER_DB_HOST,
-        port=int(TRACKER_DB_PORT),
-        name=TRACKER_DB_NAME,
+    GCP_AUTH_KEY = os.getenv("GCP_AUTH_KEY", None)
+    ENTITY_EVENT_TOPIC = os.getenv("ENTITY_EVENT_TOPIC", "events")
```

Comment:
I am not too familiar with Q topics. I set this based on the old `ENTITY_EVENT_PUBLISH_OPTIONS.subject` variable. Let me know if this is incorrect and I will fix here and the other couple places it is used.

Reply:
Removed along with publish to queue